node/settings.go

```diff
@@ -97,6 +97,7 @@ func WithBootstrappers(peers params.Bootstrappers) Option {
 }
 
 // WithRefreshRoutingTablePeriod sets custom refresh period for dht.
+// Currently, it is used to speed up tests.
```

Comment:
I meant mentioning on the test itself, so that reader of the test can understand why that option is part of the test, but this is also fine

Reply:
No action really needed, just FYI